CHANGELOG.md

```diff
@@ -19,6 +19,7 @@
 * Strimzi Access Operator installation files updated to version 0.2.0
 * New feature gate `UseConnectBuildWithBuildah` (disabled by default) for running the Connect Build feature with Buildah instead of Kaniko on Kubernetes - according to [Strimzi Proposal #114](https://github.com/strimzi/proposals/blob/main/114-use-buildah-instead-of-kaniko.md).
 * New field `spec.version` in the `KafkaConnecter` custom resource, and new `version` fields for each connector under `spec.mirrors[]` in the `KafkaMirrorMaker2` custom resource for configuring the desired version of a connector.
+* New field `.volumeAttributeClass` for `persistent-claim` type Storage in `Kafka` and `KafkaNodePool` custom resource. Enables configuring `VolumeAttributeClass` for `PersistentVolumeClaim`s available since Kubernetes v1.34.
```

Comment:
@venkatesh2090 I hate to be picky ... but could you plase fix the typo -> people will be copying it as I just did 😉. Apart from that, the actual code seems to work well.

```suggestion
* New field `.volumeAttributesClass` for `persistent-claim` type Storage in `Kafka` and `KafkaNodePool` custom resource. Enables configuring `VolumeAttributesClass` for `PersistentVolumeClaim`s available since Kubernetes v1.34.
```